src/candidates/sign/router.js

```diff
@@ -25,20 +27,23 @@ export const SIGN_ROUTER = {
       path: 'sign/success-registration',
       component: SignMessage,
       beforeEnter: (to, from, next) => {
-        if (!localStorage.getItem('successRegistration')) {
-          next({ path: '/' });
-        } else {
+        if (signStore.state.successRegistration) {
           next();
+        } else {
+          next({ path: '/' });
         }
-        // TODO fix me. Unnecessary next
-        next();
```

Comment:
This second call to `next` would always be made. Removing it interestingly results in an interesting behavior: when a user enters their credentials and clicks on "Create free profile", it creates the profile, but stays on the same page with a validation error "Email already taken". Logging the error to the console from `./src/candidates/sign/store.js#signUp` outputs:
```
Error: "Request failed with status code 400
```

Reply:
Tried to run it with both `yarn candidates` and `yarn build` + local nginx - can't reproduce.

Are there any other specific steps that you went through when you got this error?

Reply:
@ivan-kolmychek-devv Were you trying to reproduce the issue on the `master` branch?